crates/bevy_render/src/view/window.rs

```diff
@@ -40,8 +41,8 @@ impl Plugin for WindowRenderPlugin {
 }
 
 pub struct ExtractedWindow {
-    pub id: WindowId,
-    pub raw_handle: Option<RawHandleWrapper>,
+    pub id: Entity,
```

Comment:
Since we don't have kinded entities, can we at least leave a comment documenting the required components?